tests/transports-governance/conftest.py

```
@@ -0,0 +1,476 @@
+@pytest.fixture(scope="function")
+def test_data_factory():
+    """Provide test data factory"""
+    return TestDataFactory()
+
+
+class ConcurrencyTestHelper:
+    """Helper for running concurrent tests to validate atomic operations"""
+
+    @staticmethod
+    def run_concurrent_requests(
+        func, args_list: List[tuple], max_workers: int = 10
```

Ruff (0.12.2)

Missing type annotation for function argument `func`

(ANN001)